src/styles/elm-datepicker.css

```diff
@@ -134,3 +134,11 @@
 .elm-datepicker--other-month.elm-datepicker--picked {
   color: white;
 }
+
+.elm-datepicker--container.relative-table .elm-datepicker--picker {
+  @apply relative;
+}
+
+.elm-datepicker--container.relative-table .elm-datepicker--table {
+  @apply w-full;
+}
```

Comment:
line ending missing

Reply:
maybe we can add a linter rule for this?

Reply:
I'll look into a css linter we can use. I think the other linters should already report that :thinking: